cypress/component/placements/Placements.cy.tsx

```diff
@@ -248,7 +249,35 @@ describe("Placements with MFA set up", () => {
       .should("contain.text", "None provided");
   });
 
-  it("should not display Placements Subspecialty if Post does not Allow", () => {
+  it("should not display missing Placements Subspecialty if Post does not Allow", () => {
```

Comment:
These descriptions are the wrong way around I think?

Reply:
Sorry, maybe poorly phrased. The intention was to test that placements without a subspecialty (i.e. 'missing') would not have the 'Sub specialty' field displayed at all if postAllowsSubspecialty = false

Reply:
So not show 'Sub specialty: None provided' if postAllowsSubspecialty = false, basically.

Reply:
Ah, got it 🥴

Reply:
I think I snuck in the merge just before your comment 😁  - if you like I can make a PR to fix the wording if you think its worth clarifying

Reply:
Oh sorry, ok, looking at the code it is wrong - let me fix 🤦

Reply:
Actually, I'm not sure I do 😅 

e.g.
Description: "should display Placements Subspecialty even if Post does not Allow"
Mock data: `mockPlacementNoSubSpecialtyPostNotAllows`
Assertion: `cy.get('[data-cy="subSpecialty0Val"]').should("not.exist");`

The mock and the assertion match, but I can't get my head around the description of it "should display placements subspecialty" then asserting `not.exist`